README.md

```diff
@@ -352,6 +352,36 @@ CLASS_RULE_PARAM_VALIDATION = [
   - run_convert(instance, convert_to **argv) - execute the conversion for the instance with the first rule that provide the conversion (see get_convert_from_to). 'from_to' is deducted from instance. 
   - get_convert_from_to() - get the possible conversion, return [calc UUID, from, to]
 
+## CustomFilters
+This module provides the ability to define custom filters and their construction. It includes the following classes:
+* ```CustomFilterRegistryPoint```: This class is responsible for registering a specific module. 
+It should be called in the apps.py file of the respective module (see below) to enable filter construction for specific objects.
+* ```CustomFilterWizardInterface```: This interface defines the implementation details of a custom filtering wizard. 
+Each module may have different implementations for delivering the desired output.
+* ```CustomFilterWizardStorage```: This class provides a service that connects to the registered filter hub, allowing 
+retrieval of the filter construction definition for specific objects.
+
+To integrate this mechanism into an openIMIS module, follow these steps:
+* Implement the CustomFilterWizardInterface from the core module (if no custom_filters.py file exists, create one in the module) 
```

Comment:
```suggestion
* Implement the CustomFilterWizardInterface from the core module.
```
Is this file still required?

Reply:
No, this file is not required anymore according to what we discussed.